src/components/Footer/index.js

```diff
@@ -55,4 +55,4 @@ const Footer = () => {
     )
 }
 
-export default Footer
+export default React.memo(Footer)
```

Comment:
why do we need a memo on the Footer?

Reply:
I saw when i choose a lang it was rerender 4 times